src/misc.cpp

```diff
@@ -363,14 +363,15 @@ void std_aligned_free(void* ptr) {
 
 #if defined(__linux__) && !defined(__ANDROID__)
 
-void* aligned_ttmem_alloc(size_t allocSize, void*& mem) {
+void* aligned_ttmem_alloc(size_t allocSize) {
 
+  void *mem = nullptr;
   constexpr size_t alignment = 2 * 1024 * 1024; // assumed 2MB page sizes
   size_t size = ((allocSize + alignment - 1) / alignment) * alignment; // multiple of alignment
   if (posix_memalign(&mem, alignment, size))
```

Comment:
could this also use std_aligned_alloc ?

Reply:
It could, since it maps to posix_memalign(). Pushed a patch.